src/serverdlg.cpp

```diff
@@ -121,32 +121,39 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
                                       tr ( "Checked when the recorder is enabled, otherwise unchecked. "
                                            "The recorder will run when a session is in progress, if (set up correctly and) enabled." ) );
 
-    // current session directory
-    edtCurrentSessionDir->setAccessibleName ( tr ( "Current session directory text box (read-only)" ) );
-    edtCurrentSessionDir->setWhatsThis ( "<b>" + tr ( "Current Session Directory" ) + ":</b> " +
-                                         tr ( "Enabled during recording and holds the current recording session directory. "
-                                              "Disabled after recording or when the recorder is not enabled." ) );
+    // new recording
+    pbtNewRecording->setAccessibleName ( tr ( "Request new recording button" ) );
+    pbtNewRecording->setWhatsThis ( "<b>" + tr ( "New Recording" ) + ":</b> " +
+                                    tr ( "During a recording session, the button can be used to start a new recording." ) );
 
     // recorder status
     lblRecorderStatus->setAccessibleName ( tr ( "Recorder status label" ) );
     lblRecorderStatus->setWhatsThis (
         "<b>" + tr ( "Recorder Status" ) + ":</b> " + tr ( "Displays the current status of the recorder.  The following values are possible:" ) +
-        "<dl>" + "<dt>" + SREC_NOT_INITIALISED + "</dt>" + "<dd>" + tr ( "No recording directory has been set or the value is not useable" ) +
+        "<dl>" + "<dt>" + SREC_NOT_INITIALISED + "</dt>" + "<dd>" +
+        tr ( "No recording directory has been set or the value is not useable. "
+             "Check the value in the Options tab." ) +
         "</dd>" + "<dt>" + SREC_NOT_ENABLED + "</dt>" + "<dd>"
 #ifdef _WIN32
-        + tr ( "Recording has been switched off by the UI checkbox" )
+        + tr ( "Recording has been switched off by the UI checkbox." )
```

Comment:
Tiny translations change.

Reply:
With small, non-language changes like this, it is probably a good idea manually to update the `.ts` files to match. Otherwise `lupdate` creates it as a new, untranslated entry, and the person translating needs to notice that there is a similar vanished translation that could be used.

Reply:
That's a bit scary as I never look at them...  I'll have a go...

Reply:
@softins I've added a new commit with my attempt in, including the updated ts and generated qm files.

Reply:
Should I have tagged the `translation` entry at all?

Separate topic: I did notice a lot of `translation type="vanished"` entries - are those what would happen and do they need clearing up?

Reply:
OK, all entries except Chinese updated to add the ".".  I don't know - and couldn't tell from context - whether or not to add it there.